homeassistant/components/soundtouch/media_player.py

```diff
@@ -350,6 +352,21 @@ def play_media(self, media_type, media_id, **kwargs):
             else:
                 _LOGGER.warning("Unable to find preset with id %s", media_id)
 
+    def select_source(self, source):
+        """
+        Select the source to play audio from.
+
+        :param source:source to select
+        """
+        if source == "AUX":
+            _LOGGER.info("Selecting source AUX")
+            self._device.select_source_aux()
+        elif source == "Bluetooth":
+            _LOGGER.info("Selecting source Bluetooth")
+            self._device.select_source_bluetooth()
+        else:
+            _LOGGER.warning("Source not supported")
```

Comment:
Add the source to give the user more information.
```suggestion
            _LOGGER.warning("Source %s is not supported", source)
```

Reply:
Great advice, done